mobile-app/app/hooks/useNavigatorScreenOptions.ts

```diff
@@ -4,7 +4,13 @@ import { tailwind } from "@tailwind";
 import { Dimensions, Platform } from "react-native";
 import { useSafeAreaInsets } from "react-native-safe-area-context";
 
-export function useNavigatorScreenOptions(): StackNavigationOptions {
+interface Props {
+  isRounded?: boolean;
+}
+
+export function useNavigatorScreenOptions(
+  props: Props
```

Comment:
```suggestion
  props: Props = {}
```


can do this, so all others can just use `useNavigatorScreenOptions();`

Reply:
Updated this